src/airflow_docker/operator.py

```diff
@@ -232,6 +234,7 @@ def __init__(
         auto_remove=get_boolean_default("auto_remove", True),
         shm_size=None,
         provide_context=False,
+        create_host_config_kwargs=None,
```

Comment:
Should this be `additional_create_host_config_kwargs` or something?

I'm worried it's spaghetti to have this _and_ the explicit ones like `auto_remove`, `binds`, etc...

Reply:
Yep, I think so.